ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -19,16 +19,14 @@ module('Acceptance | sync | overview', function (hooks) {
   setupMirage(hooks);
 
   hooks.beforeEach(async function () {
-    syncHandlers(this.server);
     this.version = this.owner.lookup('service:version');
     this.version.features = ['Secrets Sync'];
-
-    await authPage.login();
   });
 
   module('when feature is activated', function (hooks) {
     hooks.beforeEach(async function () {
-      syncScenario(this.server);
+      syncHandlers(this.server);
+      await authPage.login();
```

Comment:
a note for other code reviewers: Angel and i moved `authPage.login` out of the top-level module and into these individual ones because `fetchActivatedFeatures` is called at the `vault.cluster` route, which happens [before the user is authenticated](https://github.com/hashicorp/vault/blob/f8b092c2014fec06f2e7758cf6bc373eb9ae1fb5/ui/app/router.js#L16-L22). as a result `auth.login` was executing and calling `fetchActivatedFeatures` before our mirage overrides like those on lines 74-95 had the chance to intercept the request.